contracts/MarketOracle.sol

```diff
@@ -12,145 +12,126 @@ interface IOracle {
 
 
 /**
- * @title Market Oracle
+ * @title Median Oracle
  *
- * @dev Provides the aggregated exchange rate data onchain using data from a whitelisted
- *      set of market source contracts.
- *      Exchange rate is the TOKEN:TARGET rate.
+ * @dev Provides aggregated values onchain using data from an authorized
```

Comment:
Grammar. How about?
Provides a value onchain that's aggregated from a whitelisted set of provider addresses.